MVT Binary data support (#5332)
visgl/deck.gl: docs/api-reference/geo-layers/mvt-layer.md

- [MVTLoader](https://loaders.gl/modules/mvt/docs/api-reference/mvt-loader)

##### `binary` (Boolean, optional)

* Default: true

> Reads data in binary format to improve performance. More information in the [performance optimization guide](/docs/developer-guide/performance.md#use-binary-data).

Reviewer: The linked documentation doesn't contain anything specific to the MVTLayer. I think the important thing to document here is whether it affects any user-facing APIs (e.g. will GeoJSON feature still be available via callbacks?)

Reviewer 2: If I just set `loadOptions: {gis: { format: 'binary' }}` will the layer auto-detect that the input is binary and do the right thing? Or I also have to set `binary` flag here?

Author: The GeoJSON layer auto-detect the input no flag required there.

Agree with @Pessimistress. I'll start adding the limitations in the next commits. A current know limitation is that binary mode is only supported if renderSubLayers is a GeoJSONLayer (the default).

- [MVTLoader](https://loaders.gl/modules/mvt/docs/api-reference/mvt-loader)

##### `binary` (Boolean, optional)

* Default: false

Use tile data in [binary format](https://github.com/visgl/loaders.gl/blob/master/modules/gis/docs/api-reference/geojson-to-binary.md) to improve performance. It removes the need for serialization and deserialization of data transferred by the worker back to the main process.

Remarks: 

> - It requires to use `GeoJsonLayer` as the `renderSubLayers` function to work.

Reviewer: ```suggestion
It requires using `GeoJsonLayer` in the `renderSubLayers` callback.
```

> - In binary format, polygons or multipolygons with holes are not supported.

Reviewer: This should be addressed in a follow up PR. Phrase it as work in progress.
